base/jaeger/deployment.yaml

```diff
@@ -28,7 +28,7 @@ spec:
               value: "9411"
             - name: MAX_TRACES
               value: "50000"
-          image: quay.io/rh_integration/jaeger-aio:latest
+          image: docker.io/jaegertracing/all-in-one:latest
```

Comment:
If we want to stay off the docker.io as much as possible, we can also use `quay.io/jaegertracing/all-in-one`, thoughts @mganisin @ficap ?

Reply:
Let's go for it, then